tests/onAfterUpdate.test.js

```diff
@@ -36,10 +35,47 @@ describe('onAfterUpdate callback', () => {
     act(() => update({}));
     expect(callback).toHaveBeenCalledTimes(2);
 
-    act(() => unmount());
+    act(() => getStore.mount()[1](false));
+    expect(callback).toHaveBeenCalledTimes(3);
+
+    act(() => update({}));
```

Comment:
Optional: we could put a comment before update twice about why this, to make it easier to understand and avoid confusions